include/envoy/stats/stats.h

```diff
@@ -99,6 +112,27 @@ class Gauge : public virtual Metric {
   virtual void set(uint64_t value) PURE;
   virtual void sub(uint64_t amount) PURE;
   virtual uint64_t value() const PURE;
+
+  // Different approaches to importing a parent's stat value. Only used for gauges; all counters
+  // simply bring in the periodic deltas.
+  enum class CombineLogic {
+    // the default; the merged result is old+new.
+    Accumulate = Flags::LogicAccumulate,
+    // import parent value only if child stat is undefined. (So, just once.)
+    OnlyImportWhenUnusedInChild = Flags::LogicUnusedOnly,
```

Comment:
Per the other comment thread, I'm thinking we should just kill this type. This is not going to work when we move the type directly into the gauge declaration, and actually isn't even how it works today (as I mentioned, stats "disappear" until the new process attaches to them). I think if we kill it will further simplify things? WDYT?

Reply:
Great, yes! Now that it's just "add" or "don't import", and doesn't appear likely to have anything more complex come back, I brought everything down to just bools.